src/common/models/essence/essence.ts

```diff
@@ -73,6 +73,10 @@ export interface EssenceJS {
   highlight?: HighlightJS;
 }
 
+export interface Context {
```

Comment:
Now that I think about it I feel like the "Context" name is too generic. So many things in Plywood and Pivot are a context to something. I think a better name here would be "EssenceContext" to go together with EssenceJS e.t.c